### ISSUE TITLE
Pay someone to be a dedicated note-taker for our recurring topical meetings

### ISSUE THREAD

**Issue by yuvipanda** [2i2c-org/team-compass#736]
### Context

We have a few recurring topical meetings (such as the product & eng meeting) where some subset of team members will be synchronously participating in discussions. Because the earth is not flat, and priorities differ, there will *always* be folks who are both interested in the content and have a vested interest in knowing what was discussed, but not actually present in the meeting. 

Note-taking of *what was said* is an important part of being inclusive in this async context. However, currently, the person taking notes is often a *participant* in the meeting as well. This does a disservice to both the note-taker (as they can not fully participate) as well as the people not in the meeting who read the notes (as notes might be incomplete). In some cases, note taking falls often onto historically marginalized people in a group, just as a reflection of the power structures we live in. In other cases, unintentionally selective note taking can provide a different picture than what actually happened.

### Proposal

We should *pay someone* to come to these recurring meetings and take notes verbatim. I would like this to be a human, and not an AI, and we should compensate them for their time appropriately. We can provide guidelines for what kinda notetaking we want. Ideally this would be someone consistent and recurring, so we can develop some kind of relationship with them too.

**Comment by jmunroe** [comment]
I agree we should have _note taking_ as a separate from those actively participating in or facillitating the meeting. This would be very worth the cost.

Would manual _note-taking_ be an example toil we want should try to avoid? 

While a human note-taker would be nice, I'd like to give some of these software tools at least some consideration too, e.g. [otter.ai](https://otter.ai/), [tldv.io](https://tldv.io/). 

I take a fair number of meetings during my work in addition to 'team meetings' so I think this technology could be worth adopting. I understand that AI based models are not great but I think there must be services (stenographers?) where a human can review and improve an automated transcription if it that was needed after the fact.

Are there other concerns with using AI based tools for note-taking that I may not be considering?

Could we first try automating the scheduling of meetings with automatic recording to produce of meetings transcripts ?  If a human still required due to a poor transcription then we could iterate on the process.

**Comment by yuvipanda** [comment]
The AI tools basically all have problems with accents outside of traditional white american male ones, and it's incredibly frustrating because some people will always consistently be 'misheard' by the AI. This is one of the other reasons I'd love for this person to be consistent - humans all sound different, and it takes a bit of time to get used to!

The live person can also ask for clarifications as it happens, which becomes difficult with automatic recordings.
